mint-auditor/src/http_api/mod.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! HTTP server for mint auditor
+
+/// Routes for the HTTP server
+mod routes;
+
+/// Service for handling HTTP requests
+mod service;
+
+/// Request and response types
+mod api_types;
```

Comment:
```suggestion
mod api_types;
mod routes;
mod service;
```

No need to document what each of them are since they have a comment inside them.